mojo/elements/utils.py

```diff
@@ -72,3 +75,41 @@ def load_mesh(
     uid = str(uuid.uuid4())
     mesh = mjcf_model.asset.add("mesh", name=f"mesh_{uid}", file=path, scale=scale)
     return mesh
+
+
+class AssetStore:
+    """Container for Mujoco assets."""
+
+    DEFAULT_CAPACITY = 32
```

Comment:
I'll assume this is the maximum number allowed in mujoco?

Reply:
Tbh I just liked the number, could have been 42 :D